pkg/build/strategies/layered/layered.go

```diff
@@ -23,6 +23,8 @@ import (
 
 const defaultDestination = "/tmp"
 
+// A Layered builder ...
+// REVIEW: improve commentary.
```

Comment:
A layered build is a build which first performs a docker build in order to inject(layer) the source/etc into the builder image, prior to running the new image with assemble.   This is necessary when the builder image does not include "sh" and "tar" as those tools are needed during normal source injection process.